classes/State.php

```diff
@@ -203,21 +203,26 @@ public function countUsed()
      *
      * @param int $idCountry Country ID
      * @param bool $active true if the state must be active
+     * @param string $orderBy order by field
      *
      * @return array|false|mysqli_result|PDOStatement|resource|null
      */
-    public static function getStatesByIdCountry($idCountry, $active = false)
+    public static function getStatesByIdCountry($idCountry, $active = false, $orderBy = null)
```

Comment:
The only issue is you can't define the order way ASC or DESC?
Should we add another parameter, or use a convention like in the webservice `name_ASC`?
Or do you think it's not useful at all?

Reply:
Very good question. I can add it in my PR if you allow me !
I would like to have the opinion of another developer
Thank you so much @jolelievre 

ping @PierreRambaud  @Progi1984

Reply:
Good idea @jolelievre